src/openms/source/ANALYSIS/OPENSWATH/MRMFeatureFinderScoring.cpp

```diff
@@ -840,7 +840,7 @@ namespace OpenMS
       if (stop_report_after_feature_ >= 0 && i >= (Size)stop_report_after_feature_) {break;}
       output.push_back(feature_list[i]);
     }
-    transition_group = transition_group_detection;
+    transition_group = transition_group_detection; // TODO why is this here?
```

Comment:
@grosenberger do you know why we are doing this? I dont quite understand the point of overwriting the `transition_group` here?

Reply:
ok great, I have changed it and now `transition_group` is `const`

Reply:
nope, seems that its important to copy back the information...